Assets/MRTK/Core/Interfaces/Devices/IMixedRealityControllerPoseSynchronizer.cs

```diff
@@ -38,6 +38,11 @@ public interface IMixedRealityControllerPoseSynchronizer : IMixedRealitySourcePo
         /// </summary>
         bool UseSourcePoseData { get; set; }
 
+        /// <summary>
+        /// Should the Transform's position use the source pose by default until the input handler events are received?
+        /// </summary>
+        bool UseSourcePoseAsFallback { get; set; }
```

Comment:
I thought an interface change was the most appropriate since we had UseSourcePoseData as a interface property, so including UseSourcePoseAsFallback seemed appropriate as well

Reply:
Not making the interface change would prevent this from being a breaking change, which is probably best at this point in the 2.x cycle.